### DISCUSSION TITLE
Noob question regarding E3V2-UBL-BLTouch-15x15-HS-v4.2.2-v1.3.3.bin

### DISCUSSION THREAD

**Discussion by ve2jpt** [Jyers/Marlin#919] [Q&A] [answered]
So a few weeks ago, I bought an Ender 3 V2 3d printer. After a few hours fiddling with it, I was able to get some pretty good results for a noob. Then my 24yo son got interested and asked if he could upgrade the hotend so he could print car parts out of nylon. So we ordered the Micro Swiss hotend and a BLTouch.

Today we installed everything and so far, it seems we did OK up to the configuration part. I have searched and have not been able to find a detailed tutorial on how to calculate the proper X, Y and Z axis offsets and how to tell the BLTouch to take these into account. 

I wanted to test print an extruder know to replace the stock one that does not fit anymore but the nozzle ends up 5-6 mm above the glass bed so it doesn't work. 

Since this is such a small piece, I did not mind having it off center but I'd really like to input the correct offset so that it prints right in the middle of the printing surface.

I'd really appreciate if someone could point me in the right direction so I can finish setting this this up and have fun printing useful things. Your help will be greatly appreciated !

**Comment by smooreace** [comment]
A quick internet search brings up at least 2 dozen videos on hot-end
changes and BLT installs, including simply measuring to find proper
starting offsets.

**Comment by Jyers** [comment, marked as answer]
For a little more clarity here, what you're probably looking for is your home offsets, these tell the printer how far the nozzle is from the corner of the bed when the printer is at it's endstops. The menu for these can be found in Control>Advanced
